app/views/course/achievements/_form.html.slim

```diff
@@ -5,6 +5,10 @@
 
   = f.input :description
 
+  = f.label t('.badge')
+  div.achievement-form = display_achievement_badge(@achievement)
+  = f.file_field :badge, class: 'upload-btn'
```

Comment:
I would think that at some point we will want to have a custom way to upload images. 

Currently now it is a primitive implementation, eg. click button to upload file, file name gets displayed, and only gets uploaded upon submitting the form.  

The other way is I remove the class first, and we do the styling later on?